assets/data/cards_pt-br.json

```
@@ -173,10 +202,12 @@
     {
       "title": "React Native",
       "description": "É um framework do Javascript baseado no React para desenvolvimento de aplicativos móveis multiplataforma."
+      "tags": ["Conceito", "Front-end"]
     },
     {
       "title": "Mobile First",
       "description": "É quando a gente primeiro pensa nos celulares e depois nos computadores na hora de fazer um site."
```

Faltou a vírugla aqui também.
```suggestion
      "description": "É quando a gente primeiro pensa nos celulares e depois nos computadores na hora de fazer um site.",
```